package.json

```diff
@@ -36,6 +36,7 @@
     "abort-controller": "^3.0.0",
     "benchmark": "^2.1.4",
     "concurrently": "^5.2.0",
+    "fastify": "^3.11.0",
```

Comment:
is this needed?

Reply:
Oh oops - its used in one of the examples and i forgot to remove it here.